src/azul/__init__.py

```diff
@@ -140,6 +141,20 @@ def data_portal_name(self):
     def dss_endpoint(self) -> str:
         return os.environ['AZUL_DSS_ENDPOINT']
 
+    @property
+    def tdr_bigquery_dataset(self) -> BigQueryDataset:
+        service, project, dataset = os.environ['AZUL_TDR_BIGQUERY_PROJECT'].split(':')
```

Comment:
```suggestion
        service, project, dataset = os.environ['AZUL_TDR_BIGQUERY_DATASET'].split(':')
```

This worries me. Did you not tests this?

Reply:
Apparently not. Adding variable access to the unit tests.